pandaserver/dataservice/EventPicker.py

```diff
@@ -366,7 +366,8 @@ def putLog(self, msg, type="debug"):
     # send email notification
     def sendEmail(self, isSucceeded, message):
```

Comment:
If I understand correctly, this function is not doing anything anymore? If so, please remove it and all the calls.